sql/core/src/test/java/test/org/apache/spark/sql/JavaDataFrameSuite.java

```diff
@@ -328,7 +328,7 @@ public void testTextLoad() {
 
   @Test
   public void testCountMinSketch() {
-    Dataset df = context.range(1000);
+    Dataset<Object> df = context.range(1000);
```

Comment:
This actually exposes a somewhat serious problem with the range function in Datasets. Can you just remove this change for now?